src/operators/UserOperations.ts

```diff
@@ -1,30 +1,53 @@
 import User, { UserInterface } from '../models/user';
+import { v4 as uuidv4 } from 'uuid';
 
-export const create = async (user: UserInterface) => User.create(user);
+export const create = async (user: UserInterface) => {
+	try {
+		user.id = uuidv4();
+		User.create(user);
+	} catch {
+		throw new Error();
```

Comment:
Don't throw an empty error here, just remove the try catch and the error that you would have caught will just propagate up.